### DISCUSSION TITLE
Please help me I am a noob

### DISCUSSION THREAD

**Discussion by dickscheney** [UltimaHoarder/UltimaScraper#1176] [General]
So for some god-forsaken reason, I can't get past the 'python start_ofd.py' step. I installed all the requirements and every time I try the command in terminal, it says 'press enter to continue' and then brings me right back to the directory line. Double clicking on the .py file just opens it up in an editor. View hidden files is enabled yet have never even seen a 'profiles' folder within the directory. Can someone please explain/help/guide me lol

![image](https://user-images.githubusercontent.com/83996312/117749583-df202980-b1c6-11eb-870f-ce3fdc3bd1f8.png)

**Comment by M-rcus** [comment]
> Execute the script with Python 3.9

That implies you're not running Python 3.9, or at least not under the `python` command.

You can try running the script with `python3.9 start_ofd.py` - if that doesn't work, then run `python --version` to figure out what version you currently have installed. If it's not `3.9.x`, then that's the problem.  

Not entirely sure what the fix would be on Windows. You might have to completely uninstall any Python version you currently have installed and then use the installer from Python.org: https://www.python.org/ftp/python/3.9.5/python-3.9.5-amd64.exe

**Comment by havingpun** [comment]
Just fixed this issue by installing the latest python, and changing the install settings. Added Python to environment variables and also selected the py launcher

**Reply by loko664** [reply]
i installed latest python and wont work either. how can i fix this?

**Comment by mattluz** [comment]
I'm currently experiencing this issue. On a Mac, running from Terminal. I have Python 3.11 installed (confirmed via Terminal) but I am still receiving the "Execute the script with Python 3.9" error. Can anyone assist?
